tests/suites/test_suite_ecp.function

```diff
@@ -50,7 +50,7 @@ inline static int mbedtls_ecp_group_cmp( mbedtls_ecp_group *grp1,
         return 1;
     if( grp1->T_size != grp2->T_size )
         return 1;
-    if( grp1->T != grp2->T )
+    if( grp1->&T != grp2->&T )
```

Comment:
This is invalid syntax in at least some of the compilers used in the CI. e.g. from Travis

```
suites/test_suite_ecp.function:53:15: error: expected identifier
    if( grp1->&T != grp2->&T )
              ^
suites/test_suite_ecp.function:53:27: error: expected identifier
    if( grp1->&T != grp2->&T )
                          ^
suites/test_suite_ecp.function:53:16: error: use of undeclared identifier 'T'
    if( grp1->&T != grp2->&T )
               ^
suites/test_suite_ecp.function:53:28: error: use of undeclared identifier 'T'
    if( grp1->&T != grp2->&T )
                           ^
4 errors generated.
make[1]: *** [Makefile:197: test_suite_ecp] Error 1
```